docs/source/markdown/options/volume.md

```diff
@@ -6,12 +6,20 @@
 
 Create a bind mount. If `-v /HOST-DIR:/CONTAINER-DIR` is specified, Podman
 bind mounts `/HOST-DIR` from the host into `/CONTAINER-DIR` in the Podman
-container. Similarly, `-v SOURCE-VOLUME:/CONTAINER-DIR` mounts the named
-volume from the host into the container. If no such named volume exists,
+container. When specifying a host path, the host path must exist. Podman will
+not create the host path and will return an error.
+
+File globbing is supported, --volume /HOST-DIR/foo\*. Not destination paths are
+not allowed. Paths matching the globs, will be mounted on the identical
+destination path within the container.
+
+Similarly, `-v SOURCE-VOLUME:/CONTAINER-DIR` mounts the named volume from the
+host into the container. If no such named volume exists,
 Podman creates one. If no source is given, the volume is created
 as an anonymously named volume with a randomly generated name, and is
 removed when the <<container|pod>> is removed via the `--rm` flag or
-the `podman rm --volumes` command.
+the `podman rm --volumes` command. When the file path glob matches no paths on
+the host, the file path flob is treated as an anonymous named volume.
```

Comment:
```suggestion
the host, the file path glob is treated as an anonymous named volume.
```

Not sure why.  I would expect Podman to error out when the glob doesn't match.  Could be a user error which will be hard to debug when an anonymous volume is created instead.